api/src/main/java/uk/gov/hmcts/payment/api/v1/controllers/PaymentDtoFactory.java

```diff
@@ -32,7 +32,7 @@ public PaymentOldDto toDto(PaymentOld paymentOld) {
     @SneakyThrows(NoSuchMethodException.class)
     private PaymentOldDto.LinkDto cancellationLink(String userId, Integer paymentId) {
         Method method = PaymentOldController.class.getMethod("cancel", String.class, Integer.class);
-        return new PaymentOldDto.LinkDto(ControllerLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
+        return new PaymentOldDto.LinkDto(WebMvcLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
```

Comment:
Codecov is complaining this line is not covered. Can you take a look?

Reply:
Hi @YasharRahvar - This is again a same issue that we faced during apportionment. We are good with the overall coverage and it is not going down.